runtime/locale/en/food.hcl

```diff
@@ -248,8 +248,6 @@ locale {
                 deteriorates = "${name(_1)}${his_owned(_1)} ${_2} deteriorates."
             }
 
-            growth = "${name(_1)}${his_owned(_1)} ${_2} enters a period of rapid growth."
```

Comment:
Was this removed unintentionally?

Reply:
Growth buff's message was moved to `buff.hcl`.